addons/mail/static/src/core/common/message.xml

```diff
@@ -84,7 +84,8 @@
                                                             'align-self-start rounded-end-3 rounded-bottom-3': !state.isEditing and !message.is_note,
                                                             'flex-grow-1': state.isEditing,
                                                             }" t-ref="body">
-                                                    <Composer t-if="state.isEditing" autofocus="true" composer="message.composer" onDiscardCallback.bind="exitEditMode" onPostCallback.bind="exitEditMode" mode="'compact'" sidebar="false"/>
+                                                    <i t-if="message.isEmpty" class="text-muted opacity-75">This message has been removed</i>
```

Comment:
This translated string is used twice, we could export it as a const and use the const in template and `inlineBody`?